api/v1alpha1/trafficmanagerbackend_types.go

```diff
@@ -75,18 +82,33 @@ type TrafficManagerEndpointStatus struct {
 	// +required
 	Name string `json:"name"`
 
+	// ResourceID is the fully qualified Azure resource Id for the resource.
+	// Ex - /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/trafficManagerProfiles/{profileName}/azureEndpoints/{name}
+	ResourceID string `json:"resourceID,omitempty"`
```

Comment:
I think this is the same as the one in the TrafficManagerProfile?  or this is the one for the endpoint (IP)?

Reply:
this one is for azure traffic manager endpoint.